app/controllers/oauth_controller.rb

```diff
@@ -20,8 +20,8 @@ def create_omniauth
       auth = request.env["omniauth.auth"]
       current_organization.oauth_provider = auth.provider
       current_organization.oauth_uid = auth.uid
-      current_organization.oauth_token = auth.credentials.token
-      current_organization.refresh_token = auth.credentials.refresh_token
+      current_organization.encrypted_oauth_token = auth.credentials.token
```

Comment:
You should set these values to the non-encrypted fields, they will be encrypted by the gem